### PR TITLE
improvement: add alternative unit names that doesn't end with S

### PR DESCRIPTION
**Pull request by larsmoa**
Add alternative spelling of units to be more resilient.

### CODE REVIEW

**Comment by larsmoa** [comment]
> Looks good, but should we also ensure resilience towards e.g. lower-case spellings of the units?

Probably a good idea, yes...

File: viewer/core/src/datamodels/cad/CadModelFactory.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export class CadModelFactory {
 
   async createModel(externalModelIdentifier: ModelIdentifier, geometryFilter?: GeometryFilter): Promise<CadNode> {
     const metadata = await this._cadModelMetadataRepository.loadData(externalModelIdentifier);
+    console.log(metadata);
```
Comment (haakonflatval-cognite) [inline viewer/core/src/datamodels/cad/CadModelFactory.ts:35]:
```suggestion
```

**Review by haakonflatval-cognite — APPROVED** [review]
Should remove the console log, otherwise looks good! :+1:

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]
Also, this was caught in the visual tests: "Uncaught (in promise) TypeError: Cannot read property 'toLowerCase' of undefined".

Seems like a null check is needed (although I don't really see how this passes the type check, which doesn't allow undefined's in the `getDistanceToMeterConversionFactor` function

**Review by haakonflatval-cognite — APPROVED** [review]

**Pull request merged**